static/app/components/events/interfaces/breadcrumbs/breadcrumbs.spec.tsx

```diff
@@ -6,8 +6,8 @@ import {textWithMarkupMatcher} from 'sentry-test/utils';
 import Breadcrumbs from 'sentry/components/events/interfaces/breadcrumbs';
 import {BreadcrumbLevelType, BreadcrumbType} from 'sentry/types/breadcrumbs';
 import {
+  useHaveSelectedProjectsSentAnyReplayEvents,
```

Comment:
When looking at the import filename, I would expect that it exports a hook named `useReplayOnboarding` -- though I don't think we have an existing convention for naming so not a blocker